orchagent/interface_rates.lua

```diff
@@ -0,0 +1,95 @@
+-- KEYS - rif IDs
```

Comment:
This is incorrect comment - https://github.com/mykolaf/sonic-sairedis/pull/2/files#diff-6d980b032eccb32c734823410a19eddfR1665